CloudScrape.cs

```diff
@@ -147,7 +148,7 @@ public CloudScrapeClient(string apiKey, string accountId)
         /// <returns></returns>
         public CloudResponse Request(string url, string method = "GET", string body = null)
         {
-            string userPassword = CreateMD5(accountId + apiKey).ToLower();
+            string userPassword = CreateMD5(accessKey).ToLower();
```

Comment:
Wouldn't it actually be more efficient to only to this MD5 hash once - instead of for every request?